packages/scipy/meta.yaml

```diff
@@ -103,10 +111,6 @@ build:
     # Input error causes "duplicate symbol" linker errors. Empty out the file.
     echo "" > scipy/sparse/linalg/_dsolve/SuperLU/SRC/input_error.c
 
-  _retain-test-patterns:
-    - "*_page_trend_test.py"
-    - "*bws_test.py"
-
```

Comment:
This was removed when testing the installation tags functionality (and I now see why these are required). I can bring this back once the conversations above have a resolution.